### DISCUSSION TITLE
What is the latest DB2 version HammerDB support?

### DISCUSSION THREAD

**Discussion by joshuajin** [TPC-Council/HammerDB#629] [General]
What is the latest version of DB2 that HammerDB supports? 
Is IBM DB2 11.5.9 specifically supported by the latest HammerDB?

**Comment by sm-shaw** [comment]
HammerDB doesn't have official support versions as such, instead we have a [Test Matrix](https://www.hammerdb.com/docs/ch01s04.html), so the Db2 builds and tests have been tested against Db2 v11.1, however this does not mean 11.5.9 won't work, just that we haven't tested it yet. 

Or in other words, being open source and community driven it is 'Db2' that is specifically supported, and we rely on the specific database user base to advise on any issues with a particular version, and we can work together to update HammerDB to support the latest database versions and features. The preferred option is of course to have a pull request submitted for any updates needed. 

For Db2 specifically, this is a good example of how support/test works, Db2 LUW is different from Db2 z/OS and we have the following issues: 
https://github.com/TPC-Council/HammerDB/issues/174
https://github.com/TPC-Council/HammerDB/issues/175
However, not having a Db2 z/OS platform to test, we are entirely reliant on someone having the interest and platform to create a PR to resolve these issues. 

Also, to note that Db2 libraries are not included in the Docker build https://github.com/TPC-Council/HammerDB/issues/404 technically of course we can do this, however we are blocked on IBM providing advice on what "certain conditions" means.

**Comment by joshuajin** [comment]
This is very helpful. Thank you Steve!
